### PR TITLE
feat: add footer

### PR DESCRIPTION
**Pull request by BlessedAmrita**
Added:
config->content->myFooterData.js
tailwind.config.js
components->myFooter->{MyFooter.jsx , MyFooter.styles.jsx}
added links for Poppins font in index.html

### CODE REVIEW

File: src/components/myFooter/MyFooter.styles.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+export const Footer = styled.div`
+  ${tw`
+    h-[774px] custom:h-[483px] bg-brown-4 font-Poppins flex flex-col justify-between  
+  `}
+`
+
+export const FooterTop = styled.div`
+  ${tw`
+    px-[24px] pt-[23px] pb-[49px] custom:h-[400.3px] custom:px-[39px] custom:py-[64px] flex justify-between flex-col gap-[47px] custom:flex-row custom:gap-[0px]
```
Comment (Srish-ty) [inline src/components/myFooter/MyFooter.styles.jsx]: custom (1450px) is too wide to add flex-col
image
make it flex-row till min 680px

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    extend: {
+      screens: {
+        custom: '1450px',
+        customSmall: '400px'
```
Comment (Srish-ty) [inline tailwind.config.js]: add intermediate screen-sizes, 680px, 920px.
you've used 'custom' breakpoint everywhere, but is too wide.

**Review by Srish-ty — CHANGES_REQUESTED** [review]

**Comment by Srish-ty** [comment]
@BlessedAmrita run `pnpm run lint:fix`

**Comment by anmol5936** [comment]
@Shurtu-gal  LGTM 🚀
Please proceed with merging

**Comment by Shurtu-gal** [comment]
@BlessedAmrita please fix formatting.

**Review by anmol5936 — DISMISSED** [review]

**Pull request closed without merging**